packages/create-app/templates/default-app/package.json.hbs

```diff
@@ -9,16 +9,16 @@
     "dev": "concurrently \"yarn start\" \"yarn start-backend\"",
     "start": "yarn workspace app start",
     "start-backend": "yarn workspace backend start",
-    "build": "lerna run build",
+    "build": "backstage-cli repo build --all",
     "build-image": "yarn workspace backend build-image",
     "tsc": "tsc",
     "tsc:full": "tsc --skipLibCheck false --incremental false",
     "clean": "backstage-cli clean && lerna run clean",
```

Comment:
so no `repo clean` yet

Reply:
Will fill in missing commands in the future :grin:

Think the next one might actually be `repo test`, as I realized we can be a lot more explicit about executing the multi-project config in that case, and also do a lookup to support `--since`.